sql/clients_daily_v6_backfill_attribution.sql

```diff
@@ -0,0 +1,33 @@
+WITH
```

Comment:
This file is a one-off for a particular backfill, yes? My assumption seeing the file is that it would correspond to a `clients_daily_v6_backfill_attribution` table. which I don't believe is the case.

For the time being, perhaps we should call this file `clients_daily_v6.backfill_attribution.sql` so that automated tooling would be able to ignore this file similar to an `foo.init.sql` file.

Reply:
this file was accidentally picked up, i need to remove it from the commit